File: help/pt_pt/exercicios_da_ficha_9/Ficha09_005/Ficha09_005.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha09_005 {

    //Main Function
    static void Main() {
        Console.Write("Limite inferior : " );
        int inf = int.Parse(Console.ReadLine());
        Console.Write("Limite superior : " );
        int sup = int.Parse(Console.ReadLine());
        while(inf <= sup ){
            if(inf % 2 != 0 ){
                Console.Write(inf + " " );
            }
            inf++;
        }
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Thu, 2022 Sep 29 12:24

File: help/pt_pt/exercicios_da_ficha_9/Ficha09_005/Ficha09_005.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha09_005 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Limite inferior : " );
        int inf = Integer.valueOf(keyb.nextLine());
        System.out.print("Limite superior : " );
        int sup = Integer.valueOf(keyb.nextLine());
        while(inf <= sup ){
            if(inf % 2 != 0 ){
                System.out.print(inf + " " );
            }
            inf++;
        }
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Thu, 2022 Sep 29 12:24
